packages/veritone-react-common/src/components/BackButton/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Put this directly into the header, you don't need to separate it. It's a simple button...

Reply:
done